### PR TITLE
GP-775 school lookup

### PR DESCRIPTION
**Pull request by lebaz20**
Add config to generate index.js that would be just imported in other apps without any extra processing needed

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import Footer from './Footer/Footer';
+import SchoolsLookUp from './SchoolsLookUp/SchoolsLookUp';
+import FileUp from './FileUp/FileUp';
+
+export {
+	Footer,
+	SchoolsLookUp,
+	FileUp,
+}
```
Comment (adamclark-dev) [inline src/components/index.js]: Missing new line

**Comment by NoelLH** [comment]
@lebaz20 I'm not sure matching some of the less standard style rules in frost-client-registration (especially lack of semicolons) is necessarily the right move. Will raise in dev forum.

**Comment by lebaz20** [comment]
@NoelLH I added back semicolons, the only thing i care about is having consistent styling across all apps, we can add them back in `frost-client-registration`

**Comment by twhiteley** [comment]
➤ Tom Whiteley commented:

Transition made by Unito

**Pull request merged**